packages/url_launcher/url_launcher/README.md

```diff
@@ -164,3 +166,25 @@ If you do this for a URL of a page containing JavaScript, make sure to pass in
 `enableJavaScript: true`, or else the launch method will not work properly. On
 iOS, the default behavior is to open all web URLs within the app. Everything
 else is redirected to the app handler.
+
+## File scheme handling
+`file:` scheme can be used on desktop platforms: `macOS`, `Linux` and `Windows`.
+
+We recommend checking first whether the directory or file exists before calling `launch`.
+
+Example:
+```dart
+var filePath = '/path/to/file';
+final Uri uri = Uri.file(filePath);
+
+if (await File(uri.toFilePath()).exists()) {
+    if (!await launch(uri.toString())) {
+      throw 'Could not launch $uri';
+    }
```

Comment:
And one more: this body is indented 4 instead of 2, making it inconsistent with the inner if's indentation of 2 (and normal Dart practice of indenting 2)